SYCL/ESIMD/kmeans/kmeans.cpp

```diff
@@ -233,9 +233,10 @@ int main(int argc, char *argv[]) {
 
   auto submitJobs = [&]() {
     // kmeans
+    nd_range<1> Range{total_threads, 1};
     auto e = q.submit([&](cl::sycl::handler &cgh) {
       cgh.parallel_for<class kMeans>(
-          GlobalRange * LocalRange, [=](nd_item<1> it) SYCL_ESIMD_KERNEL {
```

Comment:
`GlobalRange` and `LocalRange` seem to be unused now.

Reply:
Yes. Deleted.